tests/test_js_rewriting.py

```diff
@@ -65,7 +65,9 @@ def wrap_script(text: str) -> str:
 
     def __post_init__(self):
         super().__post_init__()
-        self.expected = self.wrap_script(self.expected)
+        self.expected = self.wrap_script(
+            self.expected  # pyright: ignore[reportArgumentType]
```

Comment:
Please add a getter method (something like `expected_str`) in ContentForTest which raises an error if `expected` is not a string and return `expected` otherwise ; and use it here. Otherwise the tests will begin to be full of `pyright: ignore`